src/utils/scales-utils.js

```diff
@@ -363,7 +363,13 @@ function _computeScaleDistance(values, domain, bestDistIndex, scaleFn) {
  */
 export function _getScaleDistanceAndAdjustedDomain(data, scaleObject) {
   const {attr, domain} = scaleObject;
+
   const values = getUniquePropertyValues(data, attr);
+  // Fix time scale if a data has only one value.
+  if (scaleObject.type === TIME_SCALE_TYPE && values.length === 1) {
+    values.unshift(data[0][`${scaleObject.attr}0`]);
+  }
+
```

Comment:
I am not sure.

But if we try to pass a data without x0 or y0

![screen shot 2017-06-09 at 3 16 01 pm](https://user-images.githubusercontent.com/15861257/26996424-f8d151e6-4d26-11e7-95f0-a494f1177054.png)

we get these errors:

![screen shot 2017-06-09 at 3 14 39 pm](https://user-images.githubusercontent.com/15861257/26996426-ffd962ee-4d26-11e7-9407-654c2a80a0bd.png)